modules/sync/admin-sync.php

```diff
@@ -230,4 +245,42 @@ public function __call( $func, $args ) {
 			E_USER_ERROR
 		);
 	}
+
+	/**
+	 * Returns some data (`from_post_id` and `new_lang`) from the current request.
+	 *
+	 * @since 3.7
+	 *
+	 * @param array $post_data A post array.
+	 * @return array
+	 *
+	 * @phpstan-return array{}|array{from_post_id: int<0,max>, new_lang: string}
+	 */
+	public static function get_data_from_request( array $post_data ): array {
+		if ( ! isset( $GLOBALS['pagenow'], $_GET['_wpnonce'], $_GET['from_post'], $_GET['new_lang'], $_GET['post_type'], $post_data['post_type'] ) ) {
+			return array();
+		}
+
+		if ( 'post-new.php' !== $GLOBALS['pagenow'] ) {
+			return array();
+		}
+
+		if ( ! is_string( $_GET['_wpnonce'] ) || ! is_numeric( $_GET['from_post'] ) || ! is_string( $_GET['new_lang'] ) ) {
+			return array();
+		}
```

Comment:
I don't believe these tests are useful.

Reply:
It was to prevent php warnings/errors. Done in [56b7965](https://github.com/polylang/polylang/pull/1532/commits/56b7965ca6298a1d5bed48b59863b20d47f9a06a).